letsencrypt/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 5.2.11
+
+- Use a newer, maintained Hurricane Electric plugin.
+- Note that this requires the provider name to be updated from dns-he to dns-hurricane_electric
```

Comment:
I don't think that this is strictly necessary. So with the following changes, we can make this backwards compatible.

```suggestion
```